web/src/app/users/UserContactMethodCreateDialog.js

```diff
@@ -40,18 +64,34 @@ export default function UserContactMethodCreateDialog(props) {
   })
 
   const { loading, error } = createCMStatus
-  const fieldErrs = fieldErrors(error)
   const { title = 'Create New Contact Method', subtitle } = props
 
+  let fieldErrs = fieldErrors(error)
+  if (!queryLoading && data?.users?.nodes.length > 0) {
```

Comment:
Missing `?` after nodes

```suggestion
  if (!queryLoading && data?.users?.nodes?.length > 0) {
```